sycl/test/basic_tests/image_api.cpp

```diff
@@ -13,9 +13,9 @@
 #include <detail/kernel_impl.hpp>
 #include <detail/scheduler/scheduler.hpp>
 
+#include "../helpers.hpp"
```

Comment:
What about replacing `assert` in this file?

Reply:
This is probably (cannot check at the moment) an artifact of breaking a far too large for its own good PR into smaller PRs. I will check if that is the case and repair it.